### PR TITLE
Faucet refactoring

### PR DESCRIPTION
**Pull request by vakond**
Added minor improvements & refactorings to the faucet service:

- Unique "request ID" to log entries
- Rename API request_eth_token -> request_neon
- Rename API request_erc20_tokens -> request_erc20
- Unify format of config parameters in config file and environment variables
- Manual
- Upgrade dependencies

### CODE REVIEW

File: evm_loader/faucet/Cargo.toml [hunk 1]
```diff
@@ -1,21 +1,23 @@
 [package]
 name = "faucet"
-description = "Ethereum ERC20 Token Faucet Service"
+description = "NeonLabs Faucet Service"
 authors = ["NeonLabs Maintainers <maintainers@neonlabs.org>"]
 repository = "https://github.com/neonlabsorg/neon-evm/evm_loader/faucet"
-version = "0.5.4-dev"
+version = "0.6.0-dev"
```
Comment (vasiliy-zaznobin) [inline evm_loader/faucet/Cargo.toml:6]: This field is changed after delivering a new release version. So please revert to the 0.5.4-dev

**Review by vasiliy-zaznobin — CHANGES_REQUESTED** [review]

File: evm_loader/faucet/Cargo.toml (hunk 1 above)
Comment (vakond) [reply evm_loader/faucet/Cargo.toml:6]: No problem, but it's another version after all. Would 0.5.5-dev do?

Comment (vasiliy-zaznobin) [reply evm_loader/faucet/Cargo.toml:6]: If you like to distinguish the save version please add the commit like here: https://github.com/neonlabsorg/neon-evm/blob/bfad6460a1fd1a88eaa47af53d8f3a7f0ffbaa03/evm_loader/cli/src/main.rs#L1436

Comment (vakond) [reply evm_loader/faucet/Cargo.toml]: Well, let it be old version. Who cares?

**Review by vasiliy-zaznobin — DISMISSED** [review]

**Review by vasiliy-zaznobin — APPROVED** [review]

**Review by vasiliy-zaznobin — APPROVED** [review]

**Pull request merged**